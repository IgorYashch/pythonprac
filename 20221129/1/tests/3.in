class Mult(metaclass=dump):
    def __init__(self, val):
        self.val = val

    def __call__(self, *args, **kwargs):
        result = [x * self.val for x in args]
        result += [kwargs[name] * self.val for name in kwargs]
        return result 

cls = Mult(3)
print(*cls(9, 13, 44))
print(*cls(9,a=1, b=2, c=134))