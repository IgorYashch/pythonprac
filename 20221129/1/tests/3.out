__init__: (3,), {}
__call__: (9, 13, 44), {}
27 39 132
__call__: (9,), {'a': 1, 'b': 2, 'c': 134}
27 3 6 402
